assets/src/edit-story/components/dropDown/index.js

```diff
@@ -0,0 +1,300 @@
+      opacity: 0.3;
+    `}
+
+  svg {
+    width: 28px;
+    height: 28px;
+    color: ${({ theme }) => rgba(theme.colors.fg.v1, 0.3)};
+  }
+`;
+
+const DropDownTitle = styled.span`
+  user-select: none;
+  color: ${({ theme }) => theme.colors.fg.v1};
+  font-family: ${({ theme }) => theme.fonts.label.family};
+  font-size: ${({ theme }) => theme.fonts.label.size};
+  line-height: ${({ theme }) => theme.fonts.label.lineHeight};
+  font-weight: ${({ theme }) => theme.fonts.label.weight};
+  letter-spacing: ${({ theme }) => theme.fonts.label.letterSpacing};
+`;
+
+const DropDownListWrapper = styled.div``;
+
+const DropDownList = styled.ul`
+  position: absolute;
+  top: 100%;
+  left: 0;
+  z-index: 1000;
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  float: left;
+  min-width: 160px;
+  width: 100%;
+  padding: 5px 0;
+  margin: 2px 0 0;
+  font-size: 14px;
+  text-align: left;
+  list-style: none;
+  background-color: ${({ theme }) => theme.colors.fg.v1};
+  background-clip: padding-box;
+  border-radius: 4px;
+  box-shadow: 0 6px 12px ${({ theme }) => rgba(theme.colors.bg.v0, 0.175)};
+`;
+
+const DropDownItem = styled.li.attrs({ tabIndex: '0', role: 'option' })`
+  letter-spacing: ${({ theme }) => theme.fonts.label.letterSpacing};
+  padding: 16px;
+  margin: 0;
+  font-family: ${({ theme }) => theme.fonts.label.family};
+  font-size: ${({ theme }) => theme.fonts.label.size};
+  line-height: ${({ theme }) => theme.fonts.label.lineHeight};
+  font-weight: ${({ theme }) => theme.fonts.label.weight};
+
+  &:hover {
+    background-color: ${({ theme }) => theme.colors.bg.v11};
+  }
+
+  &:focus {
+    background-color: ${({ theme }) => theme.colors.bg.v11};
+    outline: none;
+  }
+`;
+
+function DropDown({ options, value, onChange, disabled }) {
+  DropDown.wrapperRef = useRef(null);
+  DropDown.selectRef = useRef();
+  DropDown.arrayOfOptionsRefs = [];
+
+  const [openOptions, setOpenOptions] = useState(false);
+  const [focusedOption, setFocusedOption] = useState(undefined);
+  const [searchValue, setSearchValue] = useState('');
+  const activeItem = useMemo(
+    () => options.find((item) => item.value === value),
+    [value, options]
+  );
+  const toggleOptions = useCallback(() => setOpenOptions(false), []);
+
+  useFocusOut(DropDown.wrapperRef, toggleOptions);
+  useKeyDownEffect(DropDown.wrapperRef, { key: 'esc' }, toggleOptions);
+
+  const clearOptionsRefs = () => {
+    DropDown.arrayOfOptionsRefs = [];
+  };
+
+  const handleOpenOptions = (event) => {
+    switch (event.type) {
+      case 'click':
+        setOpenOptions(!openOptions);
+        setFocusedOption(document.activeElement.id);
+        break;
+      case 'keydown':
+        if (event.key === 'Enter' || event.key === ' ') {
+          _handleOpenOptions(event);
+        }
+        break;
+      default:
+    }
+  };
+
+  const _handleOpenOptions = useCallback(() => {
+    setOpenOptions(!openOptions);
+  }, [openOptions]);
```

Comment:
When opening, make sure to focus the currently selected option. As it is now, the first option is always focused even though another option was previously selected.

Reply:
Done